packages/server/tests/clients/servicesClient.ts

```diff
@@ -289,6 +292,72 @@ export default class ServicesClient {
 
         return ethers.BigNumber.from(balance.hbars.toTinybars().toString()).mul(ServicesClient.TINYBAR_TO_WEIBAR_COEF);
     }
+
+    async createHTS( args = {
+        tokenName: 'Default Name',
+        symbol: 'HTS',
+        treasuryAccountId: '0.0.2',
```

Comment:
Make this a separate account newly created with the operator public key - allowing for future modifications if needed.
In mirror node acceptance tests we eventually hit a state where a treasury account had too many tokens associated. 
Given we want these tests to run against deployed envs also we should add this support.